airbyte-integrations/connectors/source-mysql/src/main/java/io/airbyte/integrations/source/mysql/MySqlQueryUtils.java

```diff
@@ -36,16 +36,15 @@ public class MySqlQueryUtils {
 
   public record TableSizeInfo(Long tableSize, Long avgRowLength) {}
 
-  public static final String TABLE_ESTIMATE_QUERY =
-      """
-       SELECT
-         (data_length + index_length) as %s,
-         AVG_ROW_LENGTH as %s
-      FROM
-         information_schema.tables
-      WHERE
-         table_schema = '%s' AND table_name = '%s';
-       """;
+  public static final String TABLE_ESTIMATE_QUERY = """
+                                                     SELECT
+                                                       (data_length + index_length) as %s,
+                                                       AVG_ROW_LENGTH as %s
+                                                    FROM
+                                                       information_schema.tables
+                                                    WHERE
+                                                       table_schema = '%s' AND table_name = '%s';
+                                                    """;
```

Comment:
fixing whatespaces that would have been removed by the compiler (and that triggered a warning)